entries/entry04.md

```diff
@@ -1,7 +1,31 @@
 # Entry 4
-##### X/X/XX
+##### 3/8/20
+### **Sources:**
+
+I am still using the Tinkercad Arduino Board simulator. Using this simulator, I can use an artificial Arduino Board for free. This simulator is an efficient way to test my project because I dont have to spend minutes storing away physical components. The Software provies all of Arduino's gadgets and tools.  
+
+**Links:**  
+Tinkercad: https://www.tinkercad.com  
+Arduino: https://www.arduino.cc/en/Tutorial/BuiltInExamples
+Youtube: https://www.youtube.com/watch?v3=XqVmy8Vuis  
+(I used Youtube sources to help me think of a layout for my project)
+
+
+### **Engineering Design Process:**
+
+I am still in Stage 4 of my Engineering Process design. Stage 4 is the planning stage. I am still thinking about the layout of my project. Im currently using Tinkercad to make multiple layouts for the project. I must learn how to connect the arduino board to a sheet of polymer dispersed liquid crystals. Same as last time,  **my plan** now is to learn how to control voltage levels on a lightbulb so that the light it produces increases or decreases. This will teach me how to change voltage levels on the polymer dispersed liquid crystals so that the liquid crystals have different levels of transparency. I have to decide what layout and what code I want to use for my project. An issue I have dealt with is procrastination. Someties I feel like my project wnt work. I fear that the arduino m ay not apply enough voltage to make the crystals blurry. 
+
+
+### **Knowledge:**
+
+Something I am learning in class is iterating through arrays. I can implement the use of iteration in my project by making an array called "Different Voltage Levels". Every index has a different voltage. Any time I turn the toggle, the index position chages to a different voltage level. The blurryness of the glass changes based on the different voltae levels. 
+
+
+###  **Skills:**
```

Comment:
"Reflects on at least two skills developed since the previous entry." It's good to think about what you _want_ to work on. But especially reflected on how you grew (past tense).